src/app/pages/AccountDetailsPage/AccountTokenTransfersCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Card from '@mui/material/Card'
+import CardHeader from '@mui/material/CardHeader'
+import CardContent from '@mui/material/CardContent'
+import { NUMBER_OF_ITEMS_ON_SEPARATE_PAGE } from '../../config'
+import { ErrorBoundary } from '../../components/ErrorBoundary'
+import { ScrollingDiv } from '../../components/PageLayout/ScrollingDiv'
+import { CardEmptyState } from './CardEmptyState'
+import { useAccount, useAccountTokenTransfers } from './hook'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useLoaderData } from 'react-router-dom'
+import { TokenTransfers } from '../../components/Tokens/TokenTransfers'
+import { useScreenSize } from '../../hooks/useScreensize'
+
+export const accountTokenTransfersContainerId = 'transfers'
+
+export const AccountTokenTransfersCard: FC = () => {
+  const { t } = useTranslation()
+  const scope = useRequiredScopeParam()
+  const address = useLoaderData() as string
+  const { isMobile } = useScreenSize()
+
+  const { isLoading, isFetched, transfers, pagination, totalCount, isTotalCountClipped } =
+    useAccountTokenTransfers(scope, address)
+
+  const { account } = useAccount(scope, address)
+
+  return (
+    <Card>
+      <ScrollingDiv id={accountTokenTransfersContainerId}>
+        <CardHeader disableTypography component="h3" title={t('tokens.transfers')} />
+      </ScrollingDiv>
+      <CardContent>
+        <ErrorBoundary light={true}>
+          {isFetched && !transfers?.length && <CardEmptyState label={t('account.emptyTokenTransferList')} />}
+          <TokenTransfers
+            transfers={transfers}
+            ownAddress={account?.address_eth}
+            isLoading={isLoading}
+            limit={NUMBER_OF_ITEMS_ON_SEPARATE_PAGE}
+            pagination={{
+              selectedPage: pagination.selectedPage,
+              linkToPage: pagination.linkToPage,
+              totalCount,
+              isTotalCountClipped,
+              rowsPerPage: NUMBER_OF_ITEMS_ON_SEPARATE_PAGE,
+            }}
+            verbose={!isMobile}
```

Comment:
@donouwens do we really prefer less information, instead of horizontally scrollable
| "mobile" | scrollable |
| --- | --- |
| ![0d99ad56 oasis-explorer pages dev_mainnet_sapphire_address_0x643c8A8120eFB675E82C5C58d8D473f5aDd0B5e6_token-transfers](https://github.com/oasisprotocol/explorer/assets/3758846/01018e7c-ca2a-4d3b-9869-4554e91b916a) | ![0d99ad56 oasis-explorer pages dev_mainnet_sapphire_address_0x643c8A8120eFB675E82C5C58d8D473f5aDd0B5e6_token-transfers (1)](https://github.com/oasisprotocol/explorer/assets/3758846/d606b191-c5ff-4f89-ae25-cd08affd0e18) |

Reply:
I don't have an opinion here.

Reply:
You can check what we can show for example [here](https://d7428f7c.oasis-explorer.pages.dev/mainnet/sapphire/address/0xA64b0ED9789824CBdA2E0a971190EC28347BBC4e/token-transfers#transfers):  
![image](https://github.com/oasisprotocol/explorer/assets/2093792/ee03944a-4c8f-4872-b9f1-caac94686155)

The question is, what do we keep for the mobile version? Looking at the mockup, I saw no horizontal scroll bar, so now it's just the first three columns: 
![image](https://github.com/oasisprotocol/explorer/assets/2093792/3e612930-416f-44c7-bbc6-95db94651c1a)

I was hiding the other fields on purpose, so that there is no scroll, like on the visual.

But now I will un-hide them....

Reply:
You can check out the latest version (with scrolling on mobile) here: https://13e0b1b9.oasis-explorer.pages.dev/mainnet/sapphire/address/0xA64b0ED9789824CBdA2E0a971190EC28347BBC4e/token-transfers#transfers

Reply:
OK @donouwens you can check out the latest version [here](https://67f62311.oasis-explorer.pages.dev/mainnet/sapphire/address/0xA64b0ED9789824CBdA2E0a971190EC28347BBC4e/token-transfers#transfers), with all the adjustments requested above.

Reply:
OK, so since everyone is satisfied, I'm marking this as resolved.